### PR TITLE
[PO-26] Added ECR Compatibility

### PR DESCRIPTION
**Pull request by nawazxy**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,11 +64,12 @@ test_config:
 
 # Build and tag Docker image
 build: test_config
-	@echo "Building Docker Image ..."; \
```
Comment (Niveditha-Mahendra) [inline Makefile]: why these things are updated

Comment (Niveditha-Mahendra) [reply Makefile]: check this

File: Jenkinsfile [hunk 2]
```diff
@@ -3,29 +3,50 @@ def COLOR_MAP = ['SUCCESS': 'good', 'FAILURE': 'danger', 'UNSTABLE': 'warning',
...
+        //     }
+        // }
+
+        /* Authenticates with Amazon ECR Repository and retrieves access token */
+        stage("ECR Login") {
+            when {
+                environment name: 'Mergedbool', value: 'true'
+            }
             steps {
-                sh '''#!/bin/bash -le
```
Comment (Niveditha-Mahendra) [inline Jenkinsfile:30]: Can I know the reason for removing make config section?

File: Jenkinsfile [hunk 3]
```diff
@@ -36,7 +57,7 @@ pipeline {
             }
             steps {
                 sh '''#!/bin/bash -le
-                NAMESPACE=$NAMESPACE BRANCH=$BRANCH make build
+                REGISTRY=$REGISTRY NAMESPACE=$NAMESPACE BRANCH=$BRANCH make build
```
Comment (Niveditha-Mahendra) [inline Jenkinsfile:60]: This as well why registry is removed

Comment (nawazxy) [reply Jenkinsfile:60]: I added the registry variable here, I'm guessing its not needed?

**Pull request merged**